template.py

```diff
@@ -67,6 +67,11 @@ def __init__(self, logged_in, details=None, admin=False):
 
         else:
             self.entries['menu'] = [
+                {
+                    'display_name': 'Home',
```

Comment:
Nice. :) I am happy this is only there when a user is logged out. In case user is logged in, clicking the logo should be enough, no need for this to be in the bar. 👍